python/ray/services.py

```diff
@@ -977,6 +977,60 @@ def start_raylet(redis_address,
     return raylet_name
 
 
+def start_gateway(node_ip_address,
```

Comment:
Let's only support the raylet code path, so we should pass in `raylet_name` but not `local_scheduler_name` or `manager_socket_name`.

Reply:
Brutal.

J/k. Sounds good! All for less work.